src/public/js/main.js

```diff
@@ -6,6 +6,8 @@
 
   function main(CONSTANTS) {
     var socket = io();
+    var n = 255;
+    var k = 245;
```

Comment:
This was a design decision I meant to bring up in pseudo-talk but forgot. We should definitely discuss what we want to do with n and k.